Source/ToolCore/JSBind/JSBTypeScript.cpp

```diff
@@ -57,6 +65,9 @@ String JSBTypeScript::GetScriptType(JSBFunctionType* ftype)
         scriptType = "string[]";
     }
 
+    if (ftype->type_->asHeapPtrType())
+        scriptType = "any";
+
```

Comment:
Do you recall when this happens?  It again with something derived from RefCounted?